gateway/mw_go_plugin.go

```diff
@@ -139,6 +139,13 @@ func (m *GoPluginMiddleware) loadPlugin() bool {
 		m.Path = m.goPluginFromTykVersion(VERSION)
 	}
 
+	defer func() {
+		if e := recover(); e != nil {
+			err = fmt.Errorf("%v", e)
+			log.WithError(err).Error("Recovered from panic while loading Go-plugin")
```

Comment:
Should it have explicit "return false" for readability?

Reply:
I think it doesn't make sense because we call an inner function.

Reply:
I would address this as a separate PR, function could return error, and even the current return is unread in parent:

```
// global go plugins
if m.Path != "" && m.SymbolName != "" {
        m.loadPlugin() // no check, no log
        return true
}
```